src/config/dark-sites.config

```diff
@@ -772,6 +773,7 @@ sportsurge.net
 spy-cards.lubar.me
 spyware.neocities.org
 srrdb.com
+stadia.google.com
```

Comment:
I think this one should be more restrictive as the home page is not dark-by-default.

Reply:
@Gusted how to resolve this?
Specify URL for each sub-page?

Reply:
Does stadia(after login) have multiple subpages? I thought using google's logic it's mostly 1 or 2.

Reply:
```
https://stadia.google.com/home
https://stadia.google.com/store
https://stadia.google.com/explore
https://stadia.google.com/settings
https://stadia.google.com/pro
https://stadia.google.com/store/details/<long hash game, example lower>
https://stadia.google.com/store/details/68297438f4ca40828f91b31db3ce746ercp1/sku/32da1095f7c642aeb90ed11e4e7fbae9p
```

Reply:
Damn stadia. let it be for now.